pkg/dwarf/line/state_machine.go

```diff
@@ -322,6 +322,10 @@ func (lineInfo *DebugLineInfo) LineToPC(filename string, lineno int) uint64 {
 
 // PrologueEndPC returns the first PC address marked as prologue_end in the half open interval [start, end)
 func (lineInfo *DebugLineInfo) PrologueEndPC(start, end uint64) (pc uint64, file string, line int, ok bool) {
+	if lineInfo == nil {
```

Comment:
[smile, please forgive me for any offence]  
Excuse me, I have a doubt here. Program will be crashed if `lineInfo == nil`, so can not reach here.

Reply:
[nil is a valid value for a pointer receiver](https://play.golang.org/p/ARKGhiFRrdi)